libsignal/src/main/java/org/session/libsignal/crypto/MnemonicCodec.kt

```diff
@@ -94,10 +78,8 @@ class MnemonicCodec(private val loadFileContents: (String) -> String) {
         var result = ""
         val n = truncatedWordSet.size.toLong()
         // Check preconditions
-        if (words.size < 12) { throw DecodingError.InputTooShort
-        }
-        if (words.size % 3 == 0) { throw DecodingError.MissingLastWord
-        }
+        if (words.size < 12) throw DecodingError.InputTooShort
+        if (words.size % 3 == 0) throw DecodingError.MissingLastWord
```

Comment:
What a strange check... why not have too short & too long (> 12)? Mod 3 will say "missing last word" on 3/6/9 words where we're obviously not missing just the last word..

Reply:
Ah no it won't - because the < 12 will hit first. Still odd tho.

Reply:
24 words is common in crypto.

Anyway, I just fixed the formatting.